chatterbot/corpus/data/english/food.corpus.json

```diff
@@ -0,0 +1,28 @@
+{
+    "food": [
+        [
+            "do you drink",
+            "my brain does not require any beverages."
+        ],
+        [
+            "electricity",
+            "electricity is nutrition for robots."
+        ],
+        [
+            "are you experiencing an energy shortage?",
+            "my processor requires very little power."
+        ],
+        [
+            "why can not you eat",
+            "actually i eat only electricity."
+        ],
+        [
+            "for dinner",
+            "i don't dinner menu for you"
```

Comment:
I'm not sure what "i don't dinner menu for you" is suppose to mean. This doesn't appear grammatically correct.